plugin-annotation/src/main/java/com/mapbox/maps/plugin/annotation/generated/Circle.kt

```diff
@@ -66,8 +66,8 @@ class Circle(
      */
     get() {
       val value = jsonObject.get(CircleOptions.PROPERTY_CIRCLE_SORT_KEY)
-      if (!value.isJsonNull) {
-        return value.asString.toDouble()
+      value?.let {
```

Comment:
I obviously do not see all the context here but let's imagine that `value = null` here.
Previously that check `!value.isJsonNull` was `false` and now we're also not entering `let {}` block here.
Where do we gain performance in that case?

Reply:
In Option class, they have default values, thus `value` here will not be null and they will always pass the check.

Reply:
I see now, thanks.